README.md

```diff
@@ -3,7 +3,7 @@ PublicLab.org
 
 [![Code of Conduct](https://img.shields.io/badge/code-of%20conduct-green.svg)](https://publiclab.org/conduct)
 [![Build Status](https://github.com/publiclab/plots2/workflows/tests/badge.svg?branch=main)](https://github.com/publiclab/plots2/actions)
-[![first-timers-only-friendly](http://img.shields.io/badge/first--timers--only-friendly-blue.svg?style=flat-square)](https://code.publiclab.org#r=all)
++[![first-timers-only-friendly](https://img.shields.io/badge/first--timers--only-friendly-blue.svg?style=flat-square)](https://code.publiclab.org#r=all)
```

Comment:
@fannyibz kindly remove the `+`  at the beginning of the lines you changes...on the issue description the `+` represented the lines that were to be added  and the lines starting with `-` were to be remove...sorry for the confusion.